src/LoginServer/World/Character.cs

```diff
@@ -51,5 +58,15 @@ public int[] GetEquipIds()
 		{
 			return this.Equipment;
 		}
+
+		/// <summary>
+		/// Returns the equipment properties as an array.
+		/// </summary>
+		/// <returns></returns>
+		public int[] GetEquipmentProperties()
+		{
+			// TODO: This needs to return the actual properties of equipment which have variable lengths.
```

Comment:
Hm, this is probably fine for now, because the result isn't actually used, but if someone was to try to use it, it might have unexpected consequences. Maybe an empty array would be better?

Reply:
Yes an empty array would be fine. When I was fixing this for the current version i170175, I found that if these properties aren't exactly perfect, (values, not sizes), then it breaks the commander list or crashes the client.